nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/FlowController.java

```diff
@@ -2867,6 +2867,7 @@ private ProcessorStatus getProcessorStatus(final RepositoryStatusReport report,
             status.setFlowFilesSent(entry.getFlowFilesSent());
             status.setBytesSent(entry.getBytesSent());
             status.setFlowFilesRemoved(entry.getFlowFilesRemoved());
+            status.setCounters(entry.getCounters());
```

Comment:
This should be done conditionally based on `isProcessorAuthorized`. When captured for status history purposes that `Predicate` will always result in `true`.